capstone/backend/src/test/java/com/google/spanner/LibraryFunctionsTest.java

```diff
@@ -119,9 +127,39 @@ public Void run(TransactionContext transaction) throws Exception {
   }
 
   @Test
-  public void itemTypesCheck() {
+  public void itemTypesCheckOnePage() {
     List<String> actual = LibraryFunctions.getItemTypes(0);
-    List<String> expected = Arrays.asList("CEREAL", "MILK", "WATER");
+    List<String> expected =
+        Arrays.asList(
+            "CEREAL",
+            "COOKIES",
+            "FLOUR",
+            "MILK",
+            "OLIVE OIL",
+            "PENCIL",
+            "RAMEN",
+            "SHAMPOO",
+            "SODA",
+            "SUGAR");
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void allItemTypesCheck() {
+    List<String> actual = LibraryFunctions.getItemTypes();
+    List<String> expected =
+        Arrays.asList(
+            "CEREAL",
+            "COOKIES",
+            "FLOUR",
+            "MILK",
+            "OLIVE OIL",
+            "PENCIL",
+            "RAMEN",
+            "SHAMPOO",
+            "SODA",
+            "SUGAR",
+            "WATER");
```

Comment:
Can you extract the common items from this test and the test above to a constant list? And append "WATER" to that list in this test. It makes the code more readable.

Reply:
I ended up handling getting all items in the servlet instead (and kept the page functionality of the LibraryFunctions function), so I deleted this test.